nixos/modules/services/misc/docker-registry.nix

```diff
@@ -119,6 +119,7 @@ in {
 
       serviceConfig = {
         User = "docker-registry";
+        AmbientCapabilities = "cap_net_bind_service";
```

Comment:
This is only needed if ```cfg.port < 1024``` so please add a ```mkIf``` for that.

Reply:
done that, tnx, nice addition